transaction_decoder/transaction_decoder.py

```diff
@@ -279,9 +279,10 @@ def base64_decode(self, s: str) -> str:
         return base64.b64decode(s.encode("utf-8")).decode("utf-8")
 
     def hex_to_number(self, hex: str) -> int:
-        return int(hex, 16)
+        return int(hex or "00", 16)
 
     def bech32_encode(self, address: str) -> str:
         pub_key = bytes.fromhex(address)
         words = convertbits(pub_key, 8, 5)
+        assert words is not None
```

Comment:
not sure about best practices when using assert, was this just for testing/debugging? or we usually use assert in production code

Reply:
It was not for testing/debugging, it's there to fix the linter error.